tensorflow/buildspec-2-2.yml

```diff
@@ -1,143 +0,0 @@
-account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
It is good that we are removing all the old buildspecs. However, removing the entire buildspec removes the training image creation as well. Do we want to do that?